resources/aws/eks/nodeGroups.go

```diff
@@ -48,7 +48,7 @@ func newManagedNodeGroup(e aws.Environment, name string, cluster *eks.Cluster, n
 		NodeGroupNamePrefix: e.CommonNamer.DisplayName(37, pulumi.String(name), pulumi.String("ng")),
 		ScalingConfig: awsEks.NodeGroupScalingConfigArgs{
 			DesiredSize: pulumi.Int(1),
-			MaxSize:     pulumi.Int(1),
+			MaxSize:     pulumi.Int(2),
```

Comment:
Did you fill the cluster up to the point where some pods couldn’t be scheduled ?
That’s surprising given the low resource requests set on the `tracegen` pods.

Reply:
Yes I did with the default `inv create-eks` command.

Reply:
I think that it’s because `inv create-eks` disables too many node groups (all but one).
I prefer to enable an other node group by default rather than increasing the size of this node group because it would better improve the test coverage.

Reply:
It makes sense, I propose to enable the bottlerocket one by default